src/test/java/io/javalin/TestConfiguration.kt

```diff
@@ -48,7 +48,7 @@ class TestConfiguration {
             it.server {
                 Server()
             }
-            it.metricsProvider = MetricsProvider.MICROMETER
+            it.registerPlugin(MetricsPlugin())
```

Comment:
I think it's likely that we will get another metrics provider in Javalin 3, why do you want to remove the `MetricsProvider` enum?

Reply:
My initial idea was to have separate Plugins for each provider.

I reverted the removal of the enum and removed the "NONE" value, as it is not needed anymore. The enum is now a parameter of plugin.

Reply:
That's also a valid (better) approach, but then it has to be named `MicrometerPlugin()` instead.

Reply:
I removed the provider again and renamed the plugin to `MicrometerPlugin`.